modules/showheroes-bsBidAdapter.js

```diff
@@ -49,14 +49,6 @@ const converter = ortbConverter({
     }
     return imp;
   },
-  request(buildRequest, imps, bidderRequest, context) {
-    const req = buildRequest(imps, bidderRequest, context);
-    // delete user agent from oRTB, we'll get it from the header
-    (req?.device?.ua) && delete req.device['ua'];
-    // 'sua' is 2.6 standard, we operate with 2.5
-    (req?.device?.sua) && delete req.device['sua'];
-    return req;
-  },
```

Comment:
we decided to completely remove this logic instead?

So your server is okay with receiving both `ua` and `sua` ??